bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/SpectralDistribution.scala

```diff
@@ -44,174 +44,173 @@ object LibraryStar {
 
   /** Known library stars. */
   // O
-  case object O5V     extends LibraryStar("O5V", "o5v")
-  case object O8III   extends LibraryStar("O8III", "o8iii")
-  case object O9V     extends LibraryStar("O9V model", "O9V_calspec")
-  case object O95V    extends LibraryStar("O9.5V model", "O9.5V_calspec")
+  case object O5V       extends LibraryStar("O5V (300nm-6.0um)",        "o5v")
```

Comment:
@swalker2m i wonder if this may affect data serialized

Reply:
I checked the code again and `sedSpectrum` is unchanged.
However now I remember we had some exceptions in imports during the last release as:

```
edu.gemini.spModel.pio.PioParseException: GeneralError(SpectralDistribution)
at edu.gemini.spModel.pio.codec.ParamSetCodec$$anonfun$unsafeDecode$1.apply(ParamSetCodec.scala:16)
at edu.gemini.spModel.pio.codec.ParamSetCodec$$anonfun$unsafeDecode$1.apply(ParamSetCodec.scala:16)
```

This may have started at:
https://github.com/gemini-hlsw/ocs/pull/2288